commit c5a7b5aacea23f21f2def7c175c64bedbc872202
Author: Chris Wilson <chris@chris-wilson.co.uk>
Date:   Fri Jul 1 17:23:29 2016 +0100

    drm/i915: Remove debug noise on detecting fault-injection of missed interrupts
    
    Since the tests can and do explicitly check debugfs/i915_ring_missed_irqs
    for the handling of a "missed interrupt", adding it to the dmesg at INFO
    is just noise. When it happens for real, we still class it as an ERROR.
    
    Note that I have chose to remove it entirely because when we detect the
    "missed interrupt" is irrelevant and the message contains no more
    information than we glean from looking in debugfs.
    
    Signed-off-by: Chris Wilson <chris@chris-wilson.co.uk>
    Reviewed-by: Tvrtko Ursulin <tvrtko.ursulin@intel.com>
    Link: http://patchwork.freedesktop.org/patch/msgid/1467390209-3576-20-git-send-email-chris@chris-wilson.co.uk

diff --git a/drivers/gpu/drm/i915/i915_irq.c b/drivers/gpu/drm/i915/i915_irq.c
index be25b7bdacfe..53d4c8088f28 100644
--- a/drivers/gpu/drm/i915/i915_irq.c
+++ b/drivers/gpu/drm/i915/i915_irq.c
@@ -3072,9 +3072,6 @@ static unsigned kick_waiters(struct intel_engine_cs *engine)
 		if (!test_bit(engine->id, &i915->gpu_error.test_irq_rings))
 			DRM_ERROR("Hangcheck timer elapsed... %s idle\n",
 				  engine->name);
-		else
-			DRM_INFO("Fake missed irq on %s\n",
-				 engine->name);
 
 		intel_engine_enable_fake_irq(engine);
 	}